pkgs/development/tools/uniffi-bindgen/default.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, ktlint
+, yapf
+, rubocop
+, rustfmt
+, libiconv
+, makeWrapper
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "uniffi-bindgen";
+  version = "0.17.0";
+
+  src = fetchFromGitHub {
+    owner = "mozilla";
+    repo = "uniffi-rs";
+    rev = "01a9266cef151e57b2d1f98f0dbfd524fc2c72ea"; # no tag for this release yet
+    sha256 = "sha256-7/3J77xMdx8xhrlUSpBQyU5uOKH6q/AnYA8kqEMJxRQ=";
```

Upstream just pushed the tag: https://github.com/mozilla/uniffi-rs/pull/1163#issuecomment-1040353354

```suggestion
    rev = "v${version}";
    hash = "sha256-EGyJrW0U/dnKT7OWgd8LehCyvj6mxud3QWbBVyhoK4Y=";
```